docs/pages/machine-id/guides/github-actions.mdx

```diff
@@ -0,0 +1,195 @@
+the username affiliated with a node the bot will be required to access over a `tsh ssh` session:
+
+```
+$ tctl bots add github-demo --roles=access --token=github-demo –logins=username
+```
+
+## Step 3/3. Configuring GitHub Actions
+
+Now that the bot has been successfully created, github actions can succesfully function as part of the Teleport cluster.
+
+In the GitHub workflows directory of your repo, generally `/.github/workflows/`, create a new workflow YAML file,
+in this case `actionstest.yml` that will reflect the action you'd like to configure. This guide will create an action that
+will both list nodes within the cluster using `tsh ls`, as well as write the commit SHA that triggered the workflow to
+a file within the bot node. To create this configuration, use the following YAML file:
+
+```YAML
+# This is a basic workflow to help you get started with Actionsname: Demo.
+# Will take the following action whenever a push is made to the "main" branch.
+on:
+ push:
+   branches:
+     - main
+jobs:
+ demo:
+   permissions:
+# This is required or tbot will not be able to authenticate.
+     id-token: write
+     contents: read
+# The name of the action, and the Linux distro to be used to perform the required steps.
+   name: guide-demo
+   runs-on: ubuntu-latest
+   steps:
+     - name: Checkout repository
+       uses: actions/checkout@v3
+     - name: Fetch Teleport binaries
+       uses: gravitational/teleport-actions/setup@main
+       with:
+         version: 11.0.0
+     - name: Fetch credentials
+       run: >
+         tbot start
+         --join-method=github
+         --token=github-demo
+# Use the address of the auth/proxy server for your own cluster.
+         --auth-server=example.domain:443
+         --oneshot
+         --destination-dir=/opt/machine-id
+         --data-dir=/opt/machine-id-data
+     - name: List nodes
+# Enters a command from the cluster, in this case "tsh ls" using Machine ID credentials to list remote SSH nodes.
+# Ensure that the domain "authserver.domain" matches the auth/proxy server of your own cluster.
+       run: >
+         tsh -i ./opt/machine-id/identity
+         --proxy=example.domain:443
+         ls
+     - name: Write file to remote
+# Enters a command from the cluster, in this case "tsh ssh" using Machine ID credentials to gain SSH access to an SSH node, and then write the commit hash to the "github_run_log_2" file.
+# Ensure that the domain matches the auth/proxy server, that `username` matches the username of a remote SSH username, and that hostname matches an SSH host name that is a part of the Teleport cluster configured for access.
+       run: >
+         tsh -i /opt/machine-id/identity
+         --proxy example.domain:443
+         ssh username@hostname
+         "echo $GITHUB_SHA >> ~/github_run_log_2"
+```
+
+Add, commit, and push your changes to the `main` branch of the repository.
+
+Navigate to the **Actions** tab of your github repository in a web browser of your choice. Select
```

Comment:
```suggestion
Navigate to the **Actions** tab of your GitHub repository in your web browser. Select
```

Just making it a little more succinct (and fixing a GH caps issue I missed earlier).